wysadzulice/assets/js/new_campaign.js

```diff
@@ -0,0 +1,96 @@
+import jquery from 'jquery';
+import Planting from 'plantingjs';
+import { Model, View } from 'backbone';
+
+
+const NewCampaignMapsView = View.extend({
+
+  initialize: function(options) {
+    this.parent = options.parent;
+    this.engine = new Planting({
+      container: this.$el,
+      manifestoUrl: options.manifestoUrl,
+      googleApiKey: options.googleApiKey,
+      saveUrl: this.parent.saveUrl,
+      selectPanoMode: true,
+      onSelectPano: this.parent.on_select_pano.bind(this.parent),
+    });
+  },
+
+});
+
+
+const ItemView = View.extend({
+
+  initialize: function(options) {
+    this.$el.html('<img src="' + options.image + '"></img>');
+  },
+
+});
+
+const NewCampaignItemsView = View.extend({
+
+  events: {
+    'click .btn': 'save_campaign',
+  },
+
+  initialize: function(options) {
+    this.parent = options.parent;
+    this.$el.html('<div class="btn"">Zapisz</div>');
+    jquery.ajax({
```

Comment:
This needs to be separated from the view. The easiest implementation of the model could look like this:

```javascript
const CatalogModel = new Backbone.Model.extend({
  url: '/static/wysadzulice/assets/main/catalog.json'
});
```

And then in view just initialize the model and call `fetch`. Because fetch returns jqXHR (jquery wrapper on standard xhr object) you can use it as promise. 

```javascript
initialize: () {
  this.model = new CatalogModel();
  this.model
        .fetch()
        .then(this.render);
}
```

In render just use data from model.